internals/cli/export_test.go

```diff
@@ -21,16 +21,28 @@ import (
 )
 
 func RunMain() error {
-	_, ClientConfig.Socket = getEnvPaths()
 	return Run(&RunOptions{
-		ClientConfig: ClientConfig,
+		ClientConfig: newClientConfig(),
 	})
 }
 
-var ClientConfig = &client.Config{}
+var clientConfigBaseURL string
+
+func FakeClientConfigBaseURL(baseURL string) (restore func()) {
+	clientConfigBaseURL = baseURL
+	return func() {
+		clientConfigBaseURL = ""
```

Comment:
I'm aware of the pattern of saving and restoring the old value, but this broke the tests (haven't spent the time figuring out why), so I left this the same as the old restore function in `internals/cli/cli_test.go` above (which is currently the only place where `FakeClientConfigBaseURL()` is used:

```golang
s.BaseTest.AddCleanup(func() { cli.ClientConfig.BaseURL = "" })
```